package.json

```diff
@@ -27,7 +27,7 @@
     "zip:config-check": "node libs/checkZipConfigs.js;"
   },
   "devDependencies": {
-    "@api3/chains": "3.5.0",
+    "@api3/chains": "3.5.1",
```

Comment:
Why not to the [current v4.1.0](https://github.com/api3dao/chains/releases)?

Reply:
Was going to upgrade the docs and Centurion at the same time but I think it might be best to do the docs now. Was concerned about the changes to 4.x. But no time like the present.